systemtest/src/main/java/io/strimzi/systemtest/enums/CustomResourceStatus.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.enums;
+
+public enum CustomResourceStatus {
+    Ready("Ready", "True"),
+    NotReady("NotReady", "True"),
+    Warning("Warning", "True");
+
+    private final String type;
+    private final String status;
+
+    CustomResourceStatus(String type, String status) {
+        this.type = type;
+        this.status = status;
+    }
+
+    public String getType() {
+        return type;
+    }
+
+    public String getStatus() {
+        return status;
+    }
```

Comment:
Hmmm,  why did you also add the default values in the Enum? Moreover the `getStatus()` you do not use anywhere or am I missing something :) ?

Reply:
I'm using it in some tests/classes like here https://github.com/strimzi/strimzi-kafka-operator/pull/3452/files#diff-f26c29cb7f4ba062ae37c668164617baR82